pages/allProducts.tsx

```diff
@@ -1,19 +1,32 @@
-import { Product } from 'components/Product/Product';
-import { useGetProductsQuery } from 'generated/graphql';
+import { MoreProducts } from 'components/MoreProducts/MoreProducts';
+import { useGetFirstProductsQuery } from 'generated/graphql';
+import { ChangeEvent, useState } from 'react';
+
 
 export default function AllProducts() {
-	const { data } = useGetProductsQuery();
+	const productSize = 6;
+	const { data } = useGetFirstProductsQuery({
+		variables: {
+			size: productSize,
+		},
+	});
+	const [selectedValue, setSelectedValue] = useState('From A-Z');
 	if (!data) return <h2>Problem with fetching</h2>;
+	const handleChange = (e: ChangeEvent<HTMLSelectElement>) => {
+		setSelectedValue(e.target.value);
+	};
 
 	return (
 		<section>
 			<h2 className='text-center py-16 text-2xl bg-white-smoke border-y font-merriweather md:py-20 md:text-3xl'>
 				All Products
 			</h2>
 			<div className='flex flex-col items-center mb-5'>
-				<div className='flex justify-around items-center my-5 w-full max-w-screen-xl sm:justify-between text-sm md:text-base   '>
-					<select className=' py-2 px-3  bg-button-color text-xs text-white rounded hover:bg-button-color/[0.9] sm:text-sm sm:ml-5 lg:text-base'>
-						<option value='default'>Defualt sorting</option>
+				<div className='flex justify-around items-center my-5 w-full max-w-screen-xl sm:justify-between text-sm md:text-base '>
+					<select
```

Comment:
remove standard select, use component and style it.
https://www.downshift-js.com/